packages/spaces/src/lib/SpacesDisclaimer.tsx

```diff
@@ -0,0 +1,39 @@
+import { Disclaimer } from '@availity/mui-disclaimer';
+import ReactMarkdown from 'react-markdown';
+
+import { useSpaces } from './Spaces';
+
+export interface SpacesDisclaimerProps {
```

Comment:
My personal preference is to use `type` instead of `interface`. It's just preference so you can leave it if you like this better.

Reply:
fixed